storage/store.go

```diff
@@ -1267,11 +1273,9 @@ func raftEntryFormatter(data []byte) string {
 func (s *Store) updateStoreStatus() error {
 	now := s.clock.Now().WallTime
 	s.status.UpdatedAt = now
-	s.status.RaftIDs = []int64{}
-	for raftID := range s.ranges {
-		s.status.RaftIDs = append(s.status.RaftIDs, raftID)
-	}
+	s.status.RangeCount = int32(len(s.ranges))
 	key := engine.StoreStatusKey(int32(s.Ident.StoreID))
+	s.status.Stats = proto.MVCCStats(s.scanner.Stats().MVCC)
 	if err := engine.MVCCPutProto(s.engine, nil, key, proto.ZeroTimestamp, nil, s.status); err != nil {
```

Comment:
Reminder to fix how this code is writing; it should be using the key value DB client and not the engine directly.